lib/functional_mixin.js

```diff
@@ -328,6 +328,25 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  // Waits for target element to be not empty
+  testCase.waitForElementToContain = function busterFunctionalMixin_waitForElementToContain(selector, callback)
+  {
+    // check if the target element is present & and it's not empty
```

Comment:
Because `waitForElement` is used when there is no element on the page (otherwise why would you wait for it), and with `waitForElementToContain` (and others) there is use case when element and present and already resolved into jQuery object from previous operations.